util/dvsim/SimCfg.py

```diff
@@ -465,14 +473,19 @@ def gen_results_sub(items, results, fail_msgs):
                 map_full_testplan=self.map_full_testplan)
             results_str += "\n"
             self.final_total = self.testplan.final_total
-            # append link
-            self.final_total.append(self.append_result_link("Link"))
 
+        cov_total = "N/A"
         # Append coverage results of coverage was enabled.
         if self.cov and self.cov_report_deploy.status == "P":
             results_str += "\n## Coverage Results\n"
             results_str += "\n### [Coverage Dashboard](cov_report/dashboard.html)\n\n"
             results_str += self.cov_report_deploy.cov_results
+            cov_total = self.cov_report_deploy.cov_total
+
+        # append coverage total
+        self.final_total.append(cov_total)
+        # append link
+        self.final_total.append(self.append_result_link("Link"))
```

Comment:
Rename `self.final_total` to `self.results_summary` since it not no longer a singular 'total' value - it is a list of multiple things.